content/influxdb/cloud-dedicated/reference/sample-data.md

```diff
@@ -253,6 +254,96 @@ curl --request POST \
 {{% /expand %}}
 {{< /expand-wrapper >}}
 
+## Bitcoin price data
+
+The Bitcoin price sample dataset provides Bitcoin prices from
+**2023-05-01T00:00:00Z to 2023-05-15T00:00:00Z**—_[Powered by CoinDesk](https://www.coindesk.com/price/bitcoin)_.
+
+##### Time Range
+
+**2023-05-01T00:19:00Z** to **2023-05-14T23:48:00Z**
+
+##### Schema
+
+- bitcoin <em style="opacity: .5">(measurement)</em>
+  - **tags**:
+    - code
+      - EUR
+      - GBP
+      - USD
+    - crypto
+      - bitcoin
+    - description
+      - Euro
+      - British Pound Sterling
+      - United States Dollar
+    - symbol
+      - \&euro; <em style="opacity: .5">(&euro;)</em>
+      - \&pound; <em style="opacity: .5">(&pound;)</em>
+      - \&#36; <em style="opacity: .5">(&#36;)</em>
+  - **fields**
+    - price <em style="opacity: .5">(float)</em>
+
+{{< expand-wrapper >}}
+{{% expand "Write the Bitcoin sample data to InfluxDB" %}}
+
+#### Write the Bitcoin price sample data to InfluxDB
+
+Use the InfluxDB v2 or v1 API to write the Bitcoin price sample data to
+{{< cloud-name >}}.
+Replace the following in the script below:
+
+- `DATABASE_NAME`: your InfluxDB Cloud Dedicated database
+- `DATABASE_TOKEN`: a [database token](/influxdb/cloud-dedicated/admin/tokens/)
+  with sufficient permissions to the database
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[v2 API](#)
+[v1 API](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+
+{{% code-callout "DATABASE_TOKEN|DATABASE_NAME" "magenta" %}}
+```sh
+export INFLUX_HOST=http://cluster-id.influxdb.io
```

Comment:
Seems like we should either 1) export the variables outside of the script or
2) not assign variables and use the placeholders in the script.

Maybe this was my fault to begin with.